cryptography_flutter/android/build.gradle

```diff
@@ -25,6 +25,8 @@ apply plugin: 'com.android.library'
 apply plugin: 'kotlin-android'
 
 android {
+    namespace "dev.dint.cryptography_flutter"
```

Comment:
It might be wise to wrap this in a conditional compatibility with AGP <4.2.
```groovy
if (project.android.hasProperty("namespace")) {
    namespace "dev.dint.cryptography_flutter"
}
```

Reply:
So that's what that was for in other projects! Neat, thanks!